tests/spec/helpers/reporters.js

```diff
@@ -31,5 +31,8 @@ jasmine.getEnv().addReporter(
     spec: {
       displayPending: true,
     },
+    summary: {
+      displayStacktrace: true,
```

Comment:
I've added this to improve debuggability of failing tests in the legacy test suite.